pandas/core/groupby.py

```diff
@@ -1417,6 +1418,26 @@ def cumsum(self, axis=0, *args, **kwargs):
 
     @Substitution(name='groupby')
     @Appender(_doc_template)
+    def cummin(self, axis=0, *args, **kwargs):
+        """Cumulative min for each group"""
+        nv.validate_groupby_func('cummin', args, kwargs)
```

Comment:
cummin and cummax are not numpy functions I think? So in that case this line (and the args, kwargs) are not needed?